# Single detection result
float32 x          # Bounding box center x
float32 y          # Bounding box center y  
float32 width      # Bounding box width
float32 height     # Bounding box height
float32 confidence # Detection confidence [0,1]
int32 class_id     # Object class ID
string class_name  # Human readable class name